pkg/kubelet/cm/dra/plugin/noderesources.go

```diff
@@ -441,13 +456,26 @@ func (c *nodeResourcesController) sync(ctx context.Context, driverName string) e
 			continue
 		}
 
+		node, err := c.getNode()
```

Comment:
Please add a comment about why we get the node anew each time, otherwise someone might get the idea later that this should be optimized.

Reply:
Than you for the review. Added.